### PR TITLE
add cancel button on cetune status page

### PR DESCRIPTION
**Pull request by Shaoshian**
Signed-off-by: root <shianx.a.shao@intel.com>

### CODE REVIEW

File: webui/webui.py [hunk 1]
```diff
@@ -123,19 +123,31 @@ def execute(self):
...
             return "true"
         else:
             return "false"
 
+    def cancel_one(self):
+        if web.cache["tuner_thread"]:
+            pid = web.cache["tuner_thread"].pid
+            os.kill((pid+1), signal.SIGINT)
+            web.cache["cetune_status"] = "running, caught cancel request, working to close]"
+            os.system("echo 'cancel_one' > ../conf/cancel_type.conf")
```
Comment (xuechendi) [inline webui/webui.py]: Where is the codes to change the cancel_type.conf back?

Comment (Shaoshian) [inline webui/webui.py]: ?   下次execute的时候  状态就会变成execute了啊？

Comment (Shaoshian) [inline webui/webui.py]: 有三个状态  execute   cancel_one  cancel_all   execute是正常执行无cancel操作状态   cancel_one  就是cancel当前case，走的以前cetune的流程   cancel_all 就是全部cancel  这样每次到了run的时候 ，就会跳过run里面的步骤

Comment (Shaoshian) [inline webui/webui.py]: @xuechendi

Comment (xuechendi) [inline webui/webui.py]: @Shaoshian , Hmm, I see, then please change the name a bit, cancel_type is a little confusing. Maybe execute_op_type?

Comment (Shaoshian) [inline webui/webui.py]: @xuechendi   好的

Comment (xuechendi) [inline webui/webui.py]: @Shaoshian , ping me when you amend the PR, and point me a node I can test on

**Comment by Shaoshian** [comment]
@xuechendi   已经改名了   你可以去192.168.5.22:8080 上面试一下功能

**Pull request merged**